docs/content/drive.md

```diff
@@ -293,6 +293,12 @@ Controls whether files are sent to the trash or deleted
 permanently. Defaults to true, namely sending files to the trash.  Use
 `--drive-use-trash=false` to delete files permanently instead.
 
+#### --drive-use-created-date ####
+
+Use the file creation date inplace of the modification date. Defaults to false.
+Useful when moving data the your local fs and you want to use the upload/creation date
```

Comment:
There is a typo here... "moving data the your local fs"

Can you add a bit about the google photos too - I thought that was interesting.